packages/camera/camera_android_camerax/lib/src/capture_request_options.dart

```diff
@@ -112,6 +112,12 @@ class _CaptureRequestOptionsHostApiImpl extends CaptureRequestOptionsHostApi {
             throw ArgumentError(
                 'A controlAeLock value must be specified as a bool, but a $valueRuntimeType was specified.');
           }
+        case CaptureRequestKeySupportedType.controlVideoStabilizationMode:
+          if (valueRuntimeType != int) {
+            throw ArgumentError(
+                'A controlVideoStabilizationMode value must be specified as an int, but a $valueRuntimeType was specified.');
+          }
+
```

Comment:
Can you also update the test for this method? https://github.com/flutter/packages/blob/e978e06eb9207e4bd7a56b2139b2906d3150edb5/packages/camera/camera_android_camerax/test/capture_request_options_test.dart#L92-L141

Reply:
Oops, missed that one. Updated.

Thanks for bearing with my code!

Reply:
No worries there's a lot of tests. Thanks for adding it!